### PR TITLE
OSSMDOC-328: istio differences

### PR DESCRIPTION
**Pull request by neal-timpe**
- Aligned team: Service mesh
- For branches: 4.6, 4.7, 4.8
- https://issues.redhat.com/browse/OSSMDOC-328
- https://deploy-preview-33842--osdocs.netlify.app/openshift-enterprise/latest/service_mesh/v2x/ossm-vs-community.html
- Mostly style updates. Switched the order of modules and renamed one.
- SME review: knrc
- QE review: yxun
- Peer review

### CODE REVIEW

**Review by yxun — APPROVED** [review]

**Review by knrc — APPROVED** [review]

File: service_mesh/v2x/ossm-vs-community.adoc [hunk 1]
```diff
@@ -5,21 +5,19 @@ include::modules/ossm-document-attributes.adoc[]
 
 toc::[]
 
-An installation of {ProductName} differs from upstream Istio community installations in multiple ways. The modifications to {ProductName} are sometimes necessary to resolve issues, provide additional features, or to handle differences when deploying on {product-title}.
-
-The current release of {ProductName} differs from the current upstream Istio community release in the following ways:
+{ProductName} differs from an installation of Istio to provide additional features or to handle differences when deploying on OpenShift.
```
Comment (bmcelvee) [inline service_mesh/v2x/ossm-vs-community.adoc:8]: s/OpenShift/{product-title}

**Comment by bmcelvee** [comment]
One edit, otherwise LGTM!

File: service_mesh/v2x/ossm-vs-community.adoc (hunk 1 above)
Comment (neal-timpe) [reply service_mesh/v2x/ossm-vs-community.adoc]: @bmcelvee Thanks! knocked that out and squashed. Ready to merge and cp to 4.6-4.8 when you are.

**Pull request merged**